install/kubernetes/tetragon/values.yaml

```diff
@@ -124,15 +124,15 @@ tetragon:
   # As a concrete example, the following will redact all passwords passed to
   # processes with the "--password" argument:
   #
-  #   {"redact": ["--password(?:\\s+|=)(\\S*)"]}
+  #   {"redact": ["--password(?:\\s+\|=)(\\S*)"]}
```

Comment:
well this is not ideal since it eventually modify these regex (binary_regex) so that docs looks better but the direct docs on values isn't exact anymore. I'm afraid you'll have to fix the thing we use for the generation, it seems the issue already exists :/. This is a complete different task than just fixing this docs so don't feel like you have to fix this!

See:
- https://github.com/norwoodj/helm-docs/issues/118
- https://github.com/norwoodj/helm-docs/issues/171

Reply:
Thanks! You are right.
Should I add this to the Issue?

Reply:
as you wish! if you don't consider fixing this other project's issue I might take a look eventually, tell me